usaspending_api/etl/es_etl_helpers.py

```diff
@@ -276,9 +301,15 @@ def download_csv(count_sql, copy_sql, filename, job_id, skip_counts, verbose):
 
 def csv_chunk_gen(filename, chunksize, job_id):
     printf({"msg": "Opening {} (batch size = {})".format(filename, chunksize), "job": job_id, "f": "ES Ingest"})
+    # Need a specific converter to handle converting strings to correct data types (e.g. string -> array)
+    converters = {
```

Comment:
I don't know this code well, but is there somewhere static we could put this, as a constant? The values in here aren't modified within this context, although on the other hand these aren't really "settings" are business logic so I could see the case for leaving it here.

Reply:
I originally wanted to put them at the top, but since they will not be used anywhere outside of this context I stuck with being declared and used in this function.